torchvision/transforms/v2/_augment.py

```diff
@@ -135,3 +138,150 @@ def _transform(
             inpt = F.erase(inpt, **params, inplace=self.inplace)
 
         return inpt
+
+
+class _BaseMixupCutmix(Transform):
+    def __init__(self, *, alpha: float, num_categories: Optional[int] = None, labels_getter="default") -> None:
+        super().__init__()
+        self.alpha = alpha
+        self._dist = torch.distributions.Beta(torch.tensor([alpha]), torch.tensor([alpha]))
+
+        self.num_categories = num_categories
+
+        self.labels_getter = labels_getter
+        self._labels_getter = _parse_labels_getter(labels_getter)
+
+    def forward(self, *inputs):
+        inputs = inputs if len(inputs) > 1 else inputs[0]
+        flat_inputs, spec = tree_flatten(inputs)
+        needs_transform_list = self._needs_transform_list(flat_inputs)
+
+        if has_any(flat_inputs, PIL.Image.Image, datapoints.BoundingBox, datapoints.Mask):
+            raise TypeError(f"{type(self).__name__}() does not support PIL images, bounding boxes and masks.")
+
+        labels = self._labels_getter(inputs)
+        if labels is None:
+            msg = "Couldn't find a label in the inputs."
+            if self.labels_getter == "default":
+                msg = f"{msg} To overwrite the default find behavior, pass a callable for labels_getter."
```

Comment:
Maybe we can write that entire message regardless of whether "default" was passed. It would simplify the logic a bit and avoid storing `self.labels_getter`.